### PR TITLE
Notification count

### PR DESCRIPTION
**Pull request by Gnito**
Pretty much ready, but waiting comments from Janne

### CODE REVIEW

File: client/app/components/composites/AvatarDropdown/AvatarDropdown.css [hunk 1]
```diff
@@ -17,3 +17,38 @@
     }
   }
 }
+
+.avatarWithNotifications {
+  position: relative;
+  height: 100%;
+}
+
+:global(.notification-background) {
```
Comment (mporkola) [inline client/app/components/composites/AvatarDropdown/AvatarDropdown.css]: Scope this somehow, maybe with `parent >` selector

File: client/app/components/composites/AvatarDropdown/AvatarDropdown.js [hunk 2]
```diff
@@ -38,18 +39,25 @@ class AvatarDropdown extends Component {
...
       onClick: this.handleClick,
       onBlur: this.handleBlur,
       tabIndex: 0,
-      className: classNames(this.props.className, touchClass, openClass, css.avatarDropdown),
+      className: classNames(this.props.className, touchClass, openClass, css.avatarDropdown, notificationsClass),
     }, [
-      r(Avatar, this.props.avatar),
+      div({ className: css.avatarWithNotifications }, [
+        r(Avatar, this.props.avatar),
+      ].concat(notificationBadgeInArray)),
```
Comment (mporkola) [inline client/app/components/composites/AvatarDropdown/AvatarDropdown.js]: maybe the spread operator (`... notificationBadgeInArray`) could work here as well?

File: client/app/components/sections/Topbar/Topbar.js [hunk 3]
```diff
@@ -6,13 +6,17 @@ import { routes as routesProp, marketplaceContext } from '../../../utils/PropTyp
...
-import MenuMobile from '../../composites/MenuMobile/MenuMobile';
+import NotificationBadge from '../../elements/NotificationBadge/NotificationBadge';
+
+// composites
 import AvatarDropdown from '../../composites/AvatarDropdown/AvatarDropdown';
-import AddNewListingButton from '../../elements/AddNewListingButton/AddNewListingButton';
 import LoginLinks from '../../composites/LoginLinks/LoginLinks';
+import Menu from '../../composites/Menu/Menu';
+import MenuMobile from '../../composites/MenuMobile/MenuMobile';
+import SearchBar from '../../composites/SearchBar/SearchBar';
```
Comment (mporkola) [inline client/app/components/sections/Topbar/Topbar.js]: 👍 for grouping

**Comment by mporkola** [comment]
Just :shipit: if the fixes are trivial

File: client/app/components/composites/AvatarDropdown/AvatarDropdown.js (hunk 2 above)
Comment (Gnito) [inline client/app/components/composites/AvatarDropdown/AvatarDropdown.js]: Isn't spread for object literals. I'm conditionally adding stuff to child-component array.

**Pull request merged**